cmd/lightchain/run.go

```diff
@@ -107,22 +107,22 @@ func runCmd() *cobra.Command {
 				os.Exit(1)
 			}
 
-			go func() {
```

Comment:
What is the reason behind removing the go thread? is there any improvement or any different from the previous behaviour?

Reply:
yes, i moved the trap signal above so there is no need of making it async anymore as the only reason for that was to execute the trapSignal method. The reason why that was possible was due to the change of TrapSignal on Tendermint side and they made it async itself